README.md

```diff
@@ -4,7 +4,7 @@ NPS Service is an app designed to let you view all of the national parks in the
 ## 💻 Installation instructions
 1. Open project on Github
 2. Clone down the repo
-3. cd into the repo and run npm install
+3. cd into the repo and run `npm install`, `npm i @material-tailwind/react`,
```

Comment:
![Screenshot 2024-02-29 at 6 01 23 PM](https://github.com/M4-Park-Checklist/m4-parks-frontend/assets/139895703/181f90ee-a7e7-4b94-a1b5-0140196129c5)
so in the package we have it listed already that makes it a dependency so it will install automatically when you run npm install. we can try it out on a new file and try it out.